### ISSUE TITLE
Can't compile frozen facenet graph 

### ISSUE THREAD

**Issue by Dima00782** [tensorflow/tensorflow#21635]
Have I written custom code: NO
OS Platform and Distribution: Ubuntu 16.04 x86_64
TensorFlow installed from: github, from source
TensorFlow version: 1.9
Bazel version: 0.16.1
CUDA/cuDNN version: N/A
GPU model and memory: N/A
Exact command to reproduce: 
`./tfcompile --graph=frozen_20170512-110547.pb --config=frozen_20170512-110547.pbtxt --cpp_class="my_super_class" --target_features="+avx2"`
Mobile device: No

My .pbtx:
```
feed {
  id { node_name: "input" }
  shape {
    dim { size: 160 }
    dim { size: 160 }
  }
}

fetch {
  id { node_name: "embeddings" }
}
```
And I've got this:
INVALID ARGUMENTS: Unable to functionalize control flow in graph: Switch ('InceptionResnetV1/Conv2d_1a_3x3/BatchNorm/cond/Switch_1') has operands ('InceptionResnetV1/Conv2d_1a_3x3/BatchNorm/cond/Switch_1/Switch' and 'InceptionResnetV1/Conv2d_1a_3x3/BatchNorm/cond/pred_id') that have different switch depths (1 != 0)
 
What Do I do wrong?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by Dima00782** [comment]
Have I written custom code: NO
OS Platform and Distribution: Ubuntu 16.04 x86_64
TensorFlow installed from: github, from source 
TensorFlow version: 1.9
Bazel version: N/A
CUDA/cuDNN version: N/A
GPU model and memory: N/A
Exact command to reproduce: ./tfcompile --graph=frozen_20170512-110547.pb --config=frozen_20170512-110547.pbtxt --cpp_class="my_super_class" --target_features="+avx2"
Mobile device: No
